src/components/SourcesTree.js

```diff
@@ -17,6 +18,15 @@ const { showMenu } = require("./shared/menu");
 const { copyToTheClipboard } = require("../utils/clipboard");
 const { throttle } = require("../utils/utils");
 
+type CreateTree = {
```

Comment:
Should we create types for `focusedItem`, `parentMap`, `sourceTree`,...? Or is using `any` ok for now?